src/hooks/useLostFoundNotifications.ts

```
@@ -0,0 +1,281 @@
+import { createClient } from "@/lib/supabase/client";
+import { useAuthHydration } from "@/hooks/useAuthHydration";
+import { useQuery, useMutation, useQueryClient } from "@/hooks/useReactQueryReplacement";
+import { toast } from "sonner";
+import type { RealtimeChannel } from "@supabase/supabase-js";
+
+// ─── Types ────────────────────────────────────────────────────────────────────
+
+export interface LostFoundNotification {
+  id: string;
+  user_id: string;
+  type: "new_match" | "claim_received" | "claim_accepted" | "claim_rejected" | "bounty_funded" | "bounty_released" | "item_resolved" | "comment_added";
```

ESLint

[error] Replace `·"new_match"·|·"claim_received"·|·"claim_accepted"·|·"claim_rejected"·|·"bounty_funded"·|·"bounty_released"·|·"item_resolved"` with `⏎····|·"new_match"⏎····|·"claim_received"⏎····|·"claim_accepted"⏎····|·"claim_rejected"⏎····|·"bounty_funded"⏎····|·"bounty_released"⏎····|·"item_resolved"⏎···`

(prettier/prettier)